test/journeys/specs/oneOnOne/dataApi/guest.js

```diff
@@ -0,0 +1,114 @@
+
+      browserLocal.execute((localAccessToken, localToUserEmail) => {
+        const csmmDom = document.createElement('div');
+        csmmDom.setAttribute('class', 'ciscospark-widget');
+        csmmDom.setAttribute('data-toggle', 'ciscospark-space');
+        csmmDom.setAttribute('data-guest-token', localAccessToken);
+        csmmDom.setAttribute('data-to-person-email', localToUserEmail);
+        csmmDom.setAttribute('data-initial-activity', 'message');
+        document.getElementById('ciscospark-widget').appendChild(csmmDom);
+        window.loadBundle('/dist-space/bundle.js');
+      }, spock.jwt, mccoy.email);
+      browserLocal.waitForVisible(`[placeholder="Send a message to ${mccoy.displayName}"]`, 30000);
+    });
+
+    it('can load the widget for the standard user to the guest user', () => {
+      browserRemote.url('/data-api/space.html');
+
+      const title = browserRemote.getTitle();
+      assert.equal(title, 'Cisco Spark Widget Test');
+
+      browserRemote.execute((localAccessToken, localToPersonId) => {
+        const csmmDom = document.createElement('div');
+        csmmDom.setAttribute('class', 'ciscospark-widget');
+        csmmDom.setAttribute('data-toggle', 'ciscospark-space');
+        csmmDom.setAttribute('data-access-token', localAccessToken);
+        csmmDom.setAttribute('data-to-person-id', localToPersonId);
+        csmmDom.setAttribute('data-initial-activity', 'message');
+        document.getElementById('ciscospark-widget').appendChild(csmmDom);
+        window.loadBundle('/dist-space/bundle.js');
+      }, mccoy.token.access_token, spock.id);
+      browserRemote.waitForVisible(`[placeholder="Send a message to ${spock.displayName}"]`, 30000);
+    });
+
+    it('displays the regular user\'s name in title bar', () => {
+      browserLocal.waitForVisible('h1.ciscospark-title');
+      browserLocal.waitUntil(() => browserLocal.getText('h1.ciscospark-title') !== 'Loading...');
+      assert.equal(browserLocal.getText('h1.ciscospark-title'), mccoy.displayName);
+    });
+
+    describe('message widget', () => {
+      let local, remote;
+      before('create test objects for message suite', () => {
+        local = {browser: browser.select('browserLocal'), displayName: spockName, user: spock};
+        remote = {browser: browser.select('browserRemote'), displayName: mccoyName, user: mccoy};
+      });
+
+      it('sends and receives messages', () => {
+        const message = 'Oh, I am sorry, Doctor. Were we having a good time?';
+        sendMessage(local, remote, message);
+        verifyMessageReceipt(remote, local, message);
+      });
+    });
+
+    describe('meet widget', () => {
+      describe('pre call experience', () => {
+        it('has a call button', () => {
+          switchToMeet(browserLocal);
+          browserLocal.waitUntil(
+            () => browserLocal.isVisible(`${meetElements.meetWidget} ${meetElements.callButton}`),
+            1500,
```

Comment:
These kinds of tiny timeout values are part of what caused so much flakiness in the refactor I just did; please increase or remove the custom timeout value.